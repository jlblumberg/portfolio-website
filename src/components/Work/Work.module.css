.flexContainer { 
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  min-height: 100vh;
  background-color: rgb(250, 250, 250);
  padding: 0 2%;
}

.headerText {
  max-width: 90%;
  margin-top: 8%;
  margin-bottom: 5%;
}

.cardActions {
  float: left;
  margin-bottom: 8px;
  margin-left: 8px;
}

.masonryGrid {
  display: -webkit-box; /* Not needed if autoprefixing */
  display: -ms-flexbox; /* Not needed if autoprefixing */
  display: flex;
  width: auto;
}

.masonryGridColumns {
  padding-right: 20px;
  padding-left: 20px; /* gutter size */
  background-clip: padding-box;
}

/* Style your items */
.masonryGrid > div { /* change div to reference your elements you put in <Masonry> */
  margin-bottom: 20px;
}

@media (max-width: 800px) {

  .masonryGridColumns {
    padding-left: 15px; 
    padding-right: 15px; 
  }
  .masonryGridColumns > div {
    margin-bottom: 15px; /* space between items */
  }
}