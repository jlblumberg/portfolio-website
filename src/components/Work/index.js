import React from 'react';
import styles from './Work.module.css'
import { Typography, IconButton } from '@material-ui/core';
import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';

const Work = () => {

  const handleClick = () => {
    alert('add in scroll functionality later')
  }

  return(
    <div className={styles.flexContainer}>
      <div className={styles.headerText} id='header'>
        <Typography variant='h5' color='textSecondary'>MY WORK</Typography>
      </div>
      <div>
        <IconButton color='primary' id='down-button' onClick={handleClick}>
          <KeyboardArrowDownIcon fontSize='large' />
        </IconButton>
      </div>
    </div>
  )
}

export default Work;