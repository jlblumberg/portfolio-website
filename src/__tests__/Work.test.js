import React from 'react';
import { shallow } from 'enzyme';
import Work from '../components/Work'

describe('Work', () => {

  const wrapper = shallow(<Work />)

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('has a down button which takes you to the contact section', () => {
    expect(wrapper.find('#down-button').exists()).toBe(true);
  });

});