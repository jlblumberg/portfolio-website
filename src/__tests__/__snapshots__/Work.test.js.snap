// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Work renders correctly 1`] = `ShallowWrapper {}`;
